AzureMonitorAgent/ama_tst/modules/helpers.py

```diff
@@ -190,6 +195,40 @@ def check_ama_installed(ama_vers):
     return (ama_exists, ama_unique)
 
 
+# openssl connect to specific endpoint
+def check_endpt_ssl(ssl_cmd, endpoint):
```

Comment:
Small nitpick, but I'd recommend leaving out any of the connection checks you're working on (i.e. the `check_endpt_ssl()` function, the internet connection errors, etc) - just for future clarity's sake. This way we can keep it clean that this PR only has all of the installation changes in it, and the next PR you make will be all of the onboarding / connectivity changes